modules/tinymce/src/themes/silver/main/ts/ui/header/StickyHeader.ts

```diff
@@ -227,12 +227,20 @@ const getBehaviours = (editor: Editor, sharedBackstage: UiFactoryBackstageShared
                   ScrollingContext.getBoundsFrom(scrollEnv)
                 );
 
+                // When toolbar location set to the top, the y starts from the container, and the available height minuses the header height,
+                // so as you scroll the scrollable container/the page, it will dock at the top and when there's insufficient height/space, it will be hidden
+                // For toolbar location is at the bottom, we want to add the starting bounds to be the container + the header height, when there's insufficient height/space,
+                // it will be hidden, and when you scroll past the editor, it will be hidden
```

Comment:
```suggestion
                // When the toolbar location is set to the top, y is the container y and height is the available container height minus the header height.
                // This is so that as you scroll the scrollable container/the page, it will dock at the top and when there's insufficient height/space, it will be hidden.
                // When the toolbar location is set to the bottom, y is the container y plus the header height. 
                // When there's insufficient height/space, it will be hidden, and when you scroll past the editor, it will be hidden
```

Reply:
I reworded and added more info to the comments.